guillotina/db/storages/pg.py

```diff
@@ -796,11 +796,29 @@ def objects_table_name(self):
     async def open(self):
         return await self.pool.acquire(timeout=self._conn_acquire_timeout)
 
-    async def close(self, con):
+    async def _close(self, con):
         try:
-            await shield(self.pool.release(con, timeout=1))
-        except (asyncio.CancelledError, asyncio.TimeoutError, asyncpg.exceptions.ConnectionDoesNotExistError):
-            log.warning("Exception on connection close", exc_info=True)
+            await self.pool.release(con)
```

Comment:
Why not keeping the timeout with a bigger number (like 30s, 60s, ...) ? to prevent this method running indefinitely (therefore, the aiotask) is something goes bad

Reply:
sure, okay